spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -755,6 +755,27 @@ private static String sqlString(org.apache.iceberg.expressions.Literal<?> lit) {
     return null;
   }
 
+  /**
+   * Returns an Iceberg Table by its name from a Spark V2 Catalog
+   *
+   * @param spark SparkSession used for looking up catalog references and tables
+   * @param name  The multipart identifier of the Iceberg table
+   * @return an Iceberg table
+   */
+  public static org.apache.iceberg.Table loadIcebergTable(SparkSession spark, String name)
+      throws ParseException, NoSuchTableException {
+    CatalogAndIdentifier catalogAndIdentifier = catalogAndIdentifier(spark, name);
+
+    CatalogPlugin catalog = catalogAndIdentifier.catalog;
+    Preconditions.checkArgument(catalog instanceof BaseCatalog, "Catalog %s(%s) cannot be used " +
```

Comment:
Little nit but I would say "cannot be used as an Iceberg table", or "is not an Iceberg Catalog"